app/models/burrito.rb

```diff
@@ -18,4 +18,14 @@ class Burrito < ApplicationRecord
   has_many(:ratings, { :class_name => "Rating", :foreign_key => "burrito_id", :dependent => :destroy })
   belongs_to(:restaurant, { :required => false, :class_name => "Restaurant", :foreign_key => "restaurant_id", :counter_cache => true })
   has_many(:raters, { :through => :ratings, :source => :rater })
+
+  def avg_rating
+    ratings = Rating.where({ :burrito_id => self.id })
+    sum_of_ratings = 0
+    ratings.each do |rating|
+      additional_ratings = rating.rating_value
+      sum_of_ratings = additional_ratings + sum_of_ratings
+    end
```

Comment:
The [shorthand](https://www.tutorialspoint.com/ruby/ruby_operators.htm) for 

```rb
sum_of_ratings = additional_ratings + sum_of_ratings
```
is

```rb
sum_of_ratings += additional_ratings
```
<img width="711" alt="Screen Shot 2021-06-04 at 12 21 40 PM" src="https://user-images.githubusercontent.com/17581658/120840119-6bf0a580-c52f-11eb-9f15-8ca739e534bf.png">

---

EVEN BETTER

There is a `sum` method you can use to calculate the[ sum of one column for a list of records](https://stackoverflow.com/a/25721330/10481804).

This would allow you to do something like

```suggestion
    sum_of_ratings = ratings.sum(:rating_value)
```

what do you think?